go/pkg/client/tree.go

```diff
@@ -227,45 +230,65 @@ func evalParentSymlinks(execRoot, relPath string, materializeOutsideExecRoot boo
 	return evaledPathBuilder.String(), symlinks, nil
 }
 
+// loadIntermediateSymlinks inserts symlink nodes into fs.
+// If the symlink path already exists in fs, it is not overwritten, which means the first entry wins.
```

Comment:
nit: Can this be even clarified as "If the source path of the symlink already exists (e.g., "a" in a-->../afile)"

Reply:
Done